src/client/containers/PullRequestDiff/__tests__/index.js

```diff
@@ -0,0 +1,69 @@
+import chai from 'chai'
+import {
+  getData,
+} from '../index'
+import { types } from 'ducks/pullrequests/actions'
+
+const expect = chai.expect
+
+describe('getData', () => {
+  it('full data', () => {
```

Comment:
Okay. Small question - by no status do you mean that there is no status in `props`? Or that the status contains errors which are not null?